app/src/main/java/org/dhis2/data/forms/dataentry/fields/LayoutProviderImpl.kt

```diff
@@ -51,7 +49,7 @@ class LayoutProviderImpl : LayoutProvider {
                 ValueTypeRenderingType.VERTICAL_RADIOBUTTONS,
                 ValueTypeRenderingType.HORIZONTAL_CHECKBOXES,
                 ValueTypeRenderingType.VERTICAL_CHECKBOXES -> R.layout.form_option_set_selector
-                else -> R.layout.form_edit_text_custom
+                else -> R.layout.form_option_set_spinner
```

Comment:
I don't understand this change, this means that if it is text type without any rendering type works as option set?